app/views/api/v1/accounts/contact_merges/create.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.partial! 'api/v1/models/contact', formats: [:json], resource: @base_contact
```

Comment:
I created a separate JSON builder because the response of the contact_merge `created_at` is returning in ISO format. We need to have it in Unix format.

Reply:
CC: @pranavrajs @muhsin-k @sojan-official